### DISCUSSION TITLE
Truncate collection

### DISCUSSION THREAD

**Discussion by apujayed** [pocketbase/pocketbase#4615] [Q&A]
Is there any way to erase all data at once from a single collection?

**Comment by ganigeorgiev** [comment]
Not directly. See https://github.com/pocketbase/pocketbase/issues/2238.

You have couple options:

- run `sqlite3 pb_data/data.db 'DELETE FROM yourCollection'`
- create a console PocketBase command that will execute the above (_see the above link for an example_)
- duplicate your collections from the Admin UI, delete the old one, and then rename the duplicated collection to the original collection name
